setup.py

```diff
@@ -42,17 +43,21 @@
         'tests': [
             'coveralls',
             'doubles',
-            'flake8<3',  # see https://github.com/zheller/flake8-quotes/issues/29
+            'flake8',
             'flake8-quotes',
-            'mock<1.1.0',
-            'psycopg2>=2.4.0',
+            'mock',
+            'psycopg2-binary',
```

Comment:
please pin to current major, to avoid surprises in the future (>=n,<n+1)

Reply:
Since `psycopg2-binary` is a part of the `tests` dependencies and it doesn't affect regular environments, pinning it won't help to avoid any surprises.
Even worse, you will not see in CI that something became broken.

I can add some version checks into `install_patches` methods and some setting to optionally disable patching if we found that version is higher than version that `opentracing-python-instrumentation` was tested with, but I believe it should be done in a separate PR.